src/models/armTemplates.ts

```diff
@@ -1,11 +1,23 @@
 import { ServerlessAzureConfig } from "./serverless";
 
+export enum ArmResourceType {
```

Comment:
Do we need to expose this?  Each ARM resource is already a class.  Can't we use the type of class as the identifier vs adding another property to get the ARM resource type?